### DISCUSSION TITLE
Webcolor for Credits

### DISCUSSION THREAD

**Discussion by markusb68** [arendst/Tasmota#10555] [Q&A] [answered]
Hello, 
Is the color for the Credits Line (at the bottom of the web interface) hard coded?

I found this line in the delivered Html at the bottom:
`<a href='https://bit.ly/tasmota' target='_blank' style='color:#aaa;'>Tasmota 9.2.0.3 by Theo Arends</a>`

I have changed the colors with "WebColor" (background to light blue) Now the credit Line is unreadable (light grey)

**Comment by arendst** [comment]
Goal?

**Reply by markusb68** [reply]
Is it possible to use "Webcolor" to change the color of the credits?

**Comment by arendst** [comment, marked as answer]
As you noticed it's not.
